db/migrate/20170929084924_add_reasons_to_cancellations.rb

```diff
@@ -0,0 +1,5 @@
+class AddReasonsToCancellations < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
@alan Are we sure this is going to complete within 750ms?

Reply:
Should not matter much actually if this fails, but I'd increase the default timeout to make sure we don't hit the timeout as we've experienced before there is no guaranty the transaction will be rolled back properly. So I'd increase the timeout to be on the safe side.

Reply:
It's different, is a timeout to acquire the lock and a timeout per statement inside the transaction.

Rails migrations transactions use an access exclusive lock, so it will lock subsequent reads and writes to that table. If for whatever reason the lock waits on another transaction to finish before starting this timeout will kick in and ensure that reads and writes that come from http requests aren't locked for a long time and cause a downtime.

The transaction contents won't even run, and even if it does it would be safe as it does not disable DDL transactions. These transactions typically run in 1 or 2 ms so it should be fine and we typically don't have long running transactions that could prevent the lock to be acquired in the first place.

Reply:
Ok sounds perfectly fine then I was worried the timeout was for the overall DDL transaction